### DISCUSSION TITLE
Chrome Error: "A network error (such as timeout, interrupted connection or unreachable host) has occurred."

### DISCUSSION THREAD

**Discussion by fimoefive** [nss-evening-cohort-14/evening-client-side#127] [Help Me!] [answered]
Every time I click the Google Icon to sign in this error pops up in the console.

```
t {code: "auth/network-request-failed", message: "A network error (such as timeout, interrupted connection or unreachable host) has occurred.", a: null}
a: null
code: "auth/network-request-failed"
message: "A network error (such as timeout, interrupted connection or unreachable host) has occurred."
__proto__: Error
constructor: ƒ t(a,b,c)
toJSON: ƒ d()
w: ƒ ()
__proto__: Object
```

**Comment by ajawashington** [comment]
Could you completely fill out your ticket? We need to know what you researched on your own.

**Reply by ajawashington** [reply]
Try restarting your chrome and make sure your network connection is strong. It looks like an error that comes when your network is offline.

**Reply by fimoefive** [reply]
Thank you AJA for researching my error and notifying me about that. After you responded I then got a 502 error in my DOM and console. Which you pointed out that it was a server error code and any 500's code is a server error.

**Comment by fimoefive** [comment, marked as answer]
Thank you, for informing me about 500's error codes.
